test/common/http/conn_manager_utility_test.cc

```diff
@@ -35,7 +35,7 @@ namespace Http {
 
 class MockInternalAddressConfig : public Http::InternalAddressConfig {
 public:
-  MOCK_CONST_METHOD1(isInternalAddress, bool(const Network::Address::Instance&));
+  MOCK_METHOD(bool, isInternalAddress, (const Network::Address::Instance&), (const));
```

Comment:
New `MOCK_METHOD` support `override` as well, do you want to do that in this PR or in a follow up?

Reply:
yeah I was planning on doing that in a separate change